app/views/components/step_by_step_element.scala.html

```diff
@@ -0,0 +1,75 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.viewmodels.NotificationsPageSingleElement
+
+@(index: Int, element: NotificationsPageSingleElement)
+
+@isDivisibleBy2(number: Int) = @{
```

Comment:
This second `@` is necessary? I'm not sure about to `@` in one line for the view